include/deal.II/matrix_free/hanging_nodes_internal.h

```diff
@@ -63,9 +62,9 @@ namespace internal
       face_z = 1 << 5,
 
       // Element has as a constraint at * = 0 or * = fe_degree edge
-      edge_xy = 1 << 6,
-      edge_yz = 1 << 7,
-      edge_zx = 1 << 8
+      edge_x = 1 << 6,
+      edge_y = 1 << 7,
+      edge_z = 1 << 8
```

Comment:
For example, you replaced `edge_zx` with `edge_y` anywhere else, but replaced it with `edge_z` here. Does it matter how you actually set the values, or is it just an identifier (like in an enum)?

Reply:
Currently, the actual values do not mapper. But in https://github.com/dealii/dealii/pull/12781 it will change.